lib/diagnostic.md

```diff
@@ -62,5 +77,9 @@ item elements) to the body DOM node, then hides it.
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+```
+$('div').click(function() {
+    alert("clicked!");
+});
+```
```

Comment:
this works, but .click is deprecated so be careful to check the docs in the future